lib/logger.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// var exists = require('101/exists');
+// var isObject = require('101/is-object');
+var bunyan = require('bunyan');
+// var Bunyan2Loggly = require('bunyan-loggly').Bunyan2Loggly;
+
+/**
+ * Bunyan logger for ponos.
+ * @author Bryan Kendall
+ * @module ponos:logger
+ */
+module.exports = bunyan.createLogger({
+  name: 'ponos',
+  streams: getStreams()
+});
+
+// Expose get streams for unit testing
+module.exports.getStreams = getStreams;
+
+/**
+ * Streams for ponos's bunyan logger.
+ * @return {array} An array of streams for the bunyan logger
+ */
+function getStreams () {
+  var streams = [{
+    level: process.env.LOG_LEVEL_STDOUT,
+    stream: process.stdout
+  }];
+
+  // if (exists(process.env.LOGGLY_TOKEN)) {
```

Comment:
WIP stuff I assume, going to figure out a good pattern for handling logging while implementing your next project?